content/2024/_index.md

```diff
@@ -23,3 +23,17 @@ applications.
         {{</ events/registration >}}
     {{</ grid/div >}}
 {{</ grid/section-container >}}
+
+<!-- Committee section -->
+{{< events/program-committee title="Program Committee" year="2024" source="committee" displayLearnMore="false" useCarousel="false" >}}
```

Comment:
`displayLearnMore="false"` is set, but the name/titles still behave like anchor tags. Is this just an issue with the shortcode?

Reply:
Good catch! It seems like it. The anchor is always rendered regardless of the params. I'll look into fixing this.

Reply:
I'm going to merge this to prevent it from being a blocker.

I'll revisit this